lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,119 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxPageHeader from '../NxPageHeader';
-import AbstractNxPageHeader from '../../AbstractNxPageHeader/AbstractNxPageHeader';
 
-describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+fdescribe('NxPageHeader', function() {
+
+  const renderEl = rtlRenderElement(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+ it('renders a simple NxPageHeader', function() {
+    renderEl();
+    const logo = screen.getByRole('img')
+    expect(logo).toBeInTheDocument();
+    
   });
```

Comment:
One of the first tests we that we usually do is for whatever role is expected on the top-level element rendered by the component.  In this case that would be `<header>`.  The [ARIA in HTML](https://www.w3.org/TR/html-aria/#el-header) specs say that this element gets a role of `banner` unless it is included within certain other elements (which `NxPageHeader` shouldn't be).  So we should test for the `banner` role.

Reply:
> One of the first tests we that we usually do is for whatever role is expected on the top-level element rendered by the component. In this case that would be `<header>`. The [ARIA in HTML](https://www.w3.org/TR/html-aria/#el-header) specs say that this element gets a role of `banner` unless it is included within certain other elements (which `NxPageHeader` shouldn't be). So we should test for the `banner` role.

Sounds good. I updated the test here: [a8d603f](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/a8d603f405e65072936359efebff634bc5b312fc)